gfx.cpp

```diff
@@ -2309,7 +2309,7 @@ class GfxInternal
         shader_model = (shader_model != nullptr ? shader_model : dxr_device_ != nullptr ? "6_5" : "6_0");
         Program &gfx_program = programs_.insert(program);
         gfx_program.shader_model_ = shader_model;
-        gfx_program.file_path_ = program.name;
+        gfx_program.file_path_ = name;
```

Comment:
Should be `gfx_program.file_path_ = (name != nullptr) ? name : program.name`
name may be nullptr in which case we can safely use program.name as in that case it is guaranteed to be valid and set to "gfx_Program%program.handle" which will be less than `kGfxConstant_MaxNameLength`

Reply:
Fixed.